### ISSUE TITLE
Seek crashing on startup on some iOS devices

### ISSUE THREAD

**Issue by tiwane** [inaturalist/SeekReactNative#441]
**Is this a camera bug?**
If so, please head to [react-native-inat-camera](https://github.com/inaturalist/react-native-inat-camera) library and fill out a bug report there.

**Describe the bug**
Users tap on the app but it crashes before they get past the splash screen.

**To Reproduce**
1. Open the app.
2. App starts but crashes before reaching the Home screen

**Expected behavior**
App should be usable.

**Screenshots**

**Smartphone (please complete the following information):**
 - Device: iPhone SE with 128 gb of storage
 - OS: iOs 15.4.1
 - Version 2.13.6

**Additional context**

**Comment by tiwane** [comment]
Received another report of this. User says  they have an "iPhone 12 Pro Max with 15.1 iOS and I’ve got 81 GB of available storage. I think I have about 50 to 80 identified species." They said they've updated the app, so they should be using 2.14.2.

Screen recording here https://inaturalist.slack.com/archives/C9KMT4MK2/p1659385627754619

**Comment by jtklein** [comment]
Yes, I have found the native crash report I think based on the device info and version. Native crash happened in RCTImageLoader, so a network request was made to an image but errored (Crash in [RCTImageLoader _loadURLRequest:progressBlock:completionBlock:]).

Overall the crash appeared on 30 devices in the last year, so maybe not something I would give a high priority, especially since network request were involved.

**Comment by tiwane** [comment]
Got another report, which I posted to Slack [here](https://inaturalist.slack.com/archives/C9KMT4MK2/p1665705233775869). iPhone 13, iOS 15.7, 2.14.3 (252).
